mitxgraders/helpers/calc.py

```diff
@@ -478,7 +478,7 @@ def parse_algebra(self):
         expr << sum_term  # pylint: disable=pointless-statement
         self.tree = (expr + stringEnd).parseString(self.math_expr)[0]
 
-    def reduce_tree(self, handle_actions, terminal_converter=None):
```

Comment:
We're not using this, and I don't think we ever will - it's a holdover from the edX code (that wasn't using it either!)